src/khoj/routers/web_client.py

```diff
@@ -34,7 +34,8 @@ def chat_page(request: Request):
 
     @web_client.get("/config", response_class=HTMLResponse)
     def config_page(request: Request):
-        return templates.TemplateResponse("config.html", context={"request": request})
+        current_config = state.config if state.config else constants.default_config
```

Comment:
This can be simplified
```suggestion
        current_config = state.config or constants.default_config
```